coalib/coala_main.py

```diff
@@ -20,6 +20,21 @@ def do_nothing(*args):
     return True
 
 
+ALL_ACTIONS_POSSIBLE = ['Do (N)othing', '(A)pply patch', 'Add (I)gnore comment',
```

Comment:
It's possible if we import all of the actions and just use for example `ShowPatchAction().get_metadata().desc` I'll need to do this for all actions. I think it's impractical but maybe i'm wrong.